README.md

```diff
@@ -1,71 +1,96 @@
-[![Code Coverage](https://scrutinizer-ci.com/g/sculpin/sculpin/badges/coverage.png?b=develop)](https://scrutinizer-ci.com/g/sculpin/sculpin/?branch=develop)
 [![Total Downloads](https://poser.pugx.org/sculpin/sculpin/downloads.svg)](https://packagist.org/packages/sculpin/sculpin)
 
+Sculpin takes data sources such as text files (Markdown, Textile, etc.) and
+transforms them using Twig templates to produce a set of static HTML files that
+can be deployed to almost any hosting platform.
 
-Sculpin takes plain old text files (Markdown, Textile, etc.) and combines
-them with Twig templates to produce a set of static HTML files that can
-be easily deployed to almost any hosting platform.
-
-Visit [sculpin.io](http://sculpin.io) for more information.
-
-
+Visit [sculpin.io](https://sculpin.io) for more information.
 
 Documentation
 -------------
 
-Sculpin has [documentation](https://sculpin.io/documentation/) but also is
-"learn by example". If you are interested in trying out Sculpin you should fire
-up a Sculpin skeleton site to see Sculpin in action.
+Sculpin documentation can be found at [https://sculpin.io/documentation](https://sculpin.io/documentation/),
+and documentation for the Twig template  language can be found at [https://twig.symfony.com](https://twig.symfony.com/)
 
+There is also a collection of Sculpin "skeletons" to help you hit the ground 
+running with your next Sculpin website.
 
 Sculpin Skeletons
 -----------------
 
-Skeletons are starting points for a new Sculpin based site. While not strictly
-required (you can always start a site from scratch) using a skeleton is a good
-way to get a bunch of structure in place with little or no effort.
+Skeletons are starting points for a new Sculpin based site. They include the 
+basic structure of a site, such as Twig-based layout files and the supporting 
+templates for generating pagination interfaces and listing your content by 
+custom-defined categories.
+
+In Sculpin, a metadata property such as a category or tag is called a "taxonomy",
+and the Blog skeletons below also provide taxonomy layouts to help you organize 
+and categorize your content in multiple ways.
 
-Unless otherwise noted, the following skeletons are **barebones**. They will have
-minimal styles (if any at all) and are not intended to be pretty. You are
-expected to add all of the style magic yourself.
+While you can always start a site from scratch, using a skeleton is a good way
+to get a bunch of structure in place with little or no effort.
+
+Unless otherwise noted, the following skeletons are **barebones**. They have
+minimal styling and design, and are intended to help you get started with adding 
+all of your own style and flavor.
 
  * [Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton)
-   A very basic Sculpin based blog.
+   A minimal Sculpin based blog, based on an older version of Bootstrap CSS.
+ * [Tailwind Blog Skeleton](https://github.com/beryllium/sculpin-tailwind-blog-skeleton)
```

Comment:
Would we rather just move these under the org's namespace?

Reply:
That's probably the ideal way to go... :)